apps/workflow_editor/views/App.js

```diff
@@ -6,35 +6,24 @@ import React, { Component } from 'react';
 
 import AppContainer from 'common-web-ui/views/AppContainer';
 
+import { navigation } from '../config/routes';
+
 export default class App extends Component {
 
   state = {
-    title: this.title
+    navigation
   };
 
   render() {
+    document.body.style.overflow = 'hidden';
     return (
       <AppContainer
           ref="container"
-          className="app"
-          header={null}
-          title={this.state.title}
-          children={this.props.children}
-          styles={{content: {padding: 0}}} />
+          disableAppBar={true}
+          disableTabPadding={true}
```

Comment:
'default parameters' is only available in ES6 (use esnext option).<br>'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).